test/integration/compile-swagger-test.coffee

```diff
@@ -85,20 +82,26 @@ describe('compile() · Swagger', ->
       )
     )
 
-    it('is compiled with no warnings', ->
-      assert.deepEqual(compilationResult.warnings, [])
-    )
-    it('is compiled with no errors', ->
-      assert.deepEqual(compilationResult.errors, [])
+    it('produces no annotations and three transactions', ->
+      assert.jsonSchema(compilationResult, createCompilationResultSchema(
+        annotations: 0
+        transactions: 3
+      ))
     )
-    context('compiles a transaction', ->
-      it('with expected request headers', ->
-        assert.deepEqual(compilationResult.transactions[0].request.headers, {
-          'Content-Type': {value: 'application/json'}
-        })
-      )
-      it('with expected response headers', ->
-        assert.deepEqual(compilationResult.transactions[0].response.headers, {})
+    [
+      'application/json'
+      'application/xml'
+      'application/json'
```

Comment:
I think that array items that fit to 80 or 100 column length could be on one line. Just a matter of preference 🙂

Reply:
Good point. I'd fix it but given the decaf is pending...